diff --git a/P25Gateway/Makefile b/P25Gateway/Makefile
index 470d2ce..fe1d7d8 100644
--- a/P25Gateway/Makefile
+++ b/P25Gateway/Makefile
@@ -4,7 +4,7 @@ CFLAGS  = -g -O3 -Wall -std=c++0x -pthread
 LIBS    = -lpthread
 LDFLAGS = -g
 
-OBJECTS = Conf.o DMRLookup.o Log.o Mutex.o Network.o P25Gateway.o Reflectors.o StopWatch.o Thread.o Timer.o UDPSocket.o Utils.o Voice.o
+OBJECTS = Conf.o Log.o Mutex.o Network.o P25Gateway.o Reflectors.o StopWatch.o Thread.o Timer.o UDPSocket.o Utils.o Voice.o
 
 all:		P25Gateway
 
diff --git a/P25Gateway/P25Gateway.cpp b/P25Gateway/P25Gateway.cpp
index 9d220b1..5198e30 100644
--- a/P25Gateway/P25Gateway.cpp
+++ b/P25Gateway/P25Gateway.cpp
@@ -19,7 +19,6 @@
 #include "P25Gateway.h"
 #include "Reflectors.h"
 #include "StopWatch.h"
-#include "DMRLookup.h"
 #include "Network.h"
 #include "Version.h"
 #include "Thread.h"
@@ -202,9 +201,6 @@ void CP25Gateway::run()
 		reflectors.setP252DMR(m_conf.getNetworkP252DMRAddress(), m_conf.getNetworkP252DMRPort());
 	reflectors.load();
 
-	CDMRLookup* lookup = new CDMRLookup(m_conf.getLookupName(), m_conf.getLookupTime());
-	lookup->read();
-
 	CTimer inactivityTimer(1000U, m_conf.getNetworkInactivityTimeout() * 60U);
 	CTimer lostTimer(1000U, 120U);
 	CTimer pollTimer(1000U, 5U);
@@ -304,10 +300,8 @@ void CP25Gateway::run()
 
 					// If we're unlinking or changing reflectors, unlink from the current one
 					if (dstId == 9999U || reflector != NULL) {
-						std::string callsign = lookup->find(srcId);
-
 						if (currentId != 9999U) {
-							LogMessage("Unlinked from reflector %u by %s", currentId, callsign.c_str());
+							LogMessage("Unlinked from reflector %u by %d", srcId);
 
 							remoteNetwork.writeUnlink(currentAddr, currentPort);
 							remoteNetwork.writeUnlink(currentAddr, currentPort);
@@ -334,8 +328,7 @@ void CP25Gateway::run()
 						currentAddr = reflector->m_address;
 						currentPort = reflector->m_port;
 
-						std::string callsign = lookup->find(srcId);
-						LogMessage("Linked to reflector %u by %s", currentId, callsign.c_str());
+						LogMessage("Linked to reflector %u by %d", currentId, srcId);
 
 						remoteNetwork.writePoll(currentAddr, currentPort);
 						remoteNetwork.writePoll(currentAddr, currentPort);
@@ -537,7 +530,5 @@ void CP25Gateway::run()
 
 	remoteNetwork.close();
 
-	lookup->stop();
-
 	::LogFinalise();
 }
